client/league.py

```diff
@@ -34,4 +34,23 @@ async def display_best_champs(message : Message) -> None:
         bestchamps_message += ", ".join([cm.champion.name for cm in good_with])
         await message.channel.send(bestchamps_message)
         Logger.log("Outputting Best Champ")
+
+    @staticmethod
+    async def display_game_pregame_summary(message : Message) -> None:
+        summoner_name = re.compile("/pregamesummary (.*)").match(message.content).group(1)
+        summoner = Summoner(name=summoner_name, region="NA")
+        player_match_history = summoner.current_match()
+        
+        output = "```"
+        output += "{:>18} {:>14} {:>14} {:>15}".format("SUMMONER", 
+                                                                "KILLS", 
```

Comment:
What does kills here do?

Reply:
I thought I changed it, It is supposed to show the champion being played by summoner.